src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,20 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToMany;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
 public class Movie implements Cloneable {
```

Comment:
Implementing Cloneable and overriding the `clone` method in entities is not recommended. Entities in Hibernate are managed by the session and the `clone` method can cause issues with the entity's lifecycle and uniqueness. Consider removing the `clone` method and the `Cloneable` interface.